src/example/EASPortal.sol

```diff
@@ -29,7 +30,7 @@ contract EASPortal is AbstractPortal {
     AttestationRequestData data;
   }
 
-  bytes32 private relationshipSchemaId = 0x89bd76e17fd84df8e1e448fa1b46dd8d97f7e8e806552b003f8386a5aebcb9f0;
+  bytes32 private _relationshipSchemaId = 0x89bd76e17fd84df8e1e448fa1b46dd8d97f7e8e806552b003f8386a5aebcb9f0;
```

Comment:
Is it better to keep it without _ just to keep it consistent? Not a big issue if we keep _ too.

Reply:
I've been thinking about it...
As it is an example, I consider it lives out of our codebase in a way, so we might as well do things properly there, wdyt?